src/main/java/com/box/sdk/BoxDeveloperEditionAPIConnection.java

```diff
@@ -201,19 +209,20 @@ public void refresh() {
         this.getRefreshLock().writeLock().unlock();
     }
 
-    String jwtConstructAssertion() {
+    private String constructJWTAssertion() {
         JwtClaims claims = new JwtClaims();
         claims.setIssuer(this.getClientID());
-        claims.setAudience(this.getTokenURL());
-        claims.setExpirationTimeMinutesInTheFuture(0.2f);
+        claims.setAudience("https://api.box.com/oauth2/token");
```

Comment:
Is there a reason why this needs to be hard-coded and not configurable? Can we add a comment explaining why?